Source/Particles/ElementaryProcess/QEDInternals/QedWrapperCommons.H

```diff
@@ -6,16 +6,19 @@
  */
 #ifndef WARPX_amrex_qed_wrapper_commons_h_
 #define WARPX_amrex_qed_wrapper_commons_h_
-
 /*
  * This header contains some common #define directives
  * used by the QED library wrappers and related
- * components.
+ * components. It also defines a thin wrapper around
```

Comment:
Not 100% sure why you define a wrapper for DeviceVector - you could also define a free standing function for `amrex::Gpu::Device::streamSynchronize();` and call that?

Reply:
I will change this comment. You are right.